install_scripts/templates/kubernetes/init.sh

```diff
@@ -81,6 +81,35 @@ set -e
 {% include 'common/firewall.sh' %}
 
 initKubeadmConfig() {
+    local kubeadmVersion=$(getKubeadmVersion)
+    semverParse "$kubeadmVersion"
+
+    # don't overwrite an alpha3 config created by kubeadm 1.12
+    if [ "$minor" -gt "12" ]; then
```

Comment:
what if it equals 12?

Reply:
It's intentionally left untouched because then it would need to be v1alpha3 and we don't have code to produce config for that version because we don't pin to 1.12